modules/utilityFunction.js

```diff
@@ -0,0 +1,127 @@
+                  </a>
+                  <a class="contact-link" target="_blank" href="${contact.instagramLink}">
+                    <i class="fab fa-instagram-square fa-fw fa-2xl"></i>
+                  </a>
+                </div>
+              </div>`;
+  }
+
+  const contactWrapper = document.querySelector('.contact-wrapper');
+
+  contactWrapper.innerHTML = contactInstanceFromContact.profile.map((contact) => loopingContacts(contact)).join('');
+}
+
+export function addBookButton(bookInstanceFromBookStore) {
+  const titleInput = document.querySelector('#title-input');
+  const authorInput = document.querySelector('#author-input');
+  const addButton = document.querySelector('#add-book-button');
+
+  addButton.addEventListener('click', (e) => {
+    e.preventDefault();
+    let newid;
+    if (bookInstanceFromBookStore.store.length === 0) {
+      newid = 1;
+    } else {
+      newid = bookInstanceFromBookStore.store[bookInstanceFromBookStore.store.length - 1].id + 1;
+    }
+    const newBook = {
+      id: newid,
+      title: titleInput.value,
+      author: authorInput.value,
+    };
+    bookInstanceFromBookStore.add(newBook);
+    loadBookslist(bookInstanceFromBookStore);
+    titleInput.value = '';
+    authorInput.value = '';
+    localStorage.setItem('bookCollection', JSON.stringify(bookInstanceFromBookStore.store));
+  });
+}
+
+export function spaExperience() {
+  const navLinks = document.querySelectorAll('.links');
+
+  navLinks.forEach((link) => {
+    link.addEventListener('click', () => {
+      const dataLink = link.getAttribute('data-page');
+      const listBookSection = document.querySelector('.list-book-section');
+      const newBookSection = document.querySelector('.new-book-section');
+      const contactUsSection = document.querySelector('.contact-us-section');
+
+      switch (dataLink) {
+        case 'list':
+          listBookSection.setAttribute('data-visible', true);
+          newBookSection.setAttribute('data-visible', false);
+          contactUsSection.setAttribute('data-visible', false);
+          break;
+        case 'add-new':
+          newBookSection.setAttribute('data-visible', true);
+          listBookSection.setAttribute('data-visible', false);
+          contactUsSection.setAttribute('data-visible', false);
+          break;
+        case 'contacts':
+          contactUsSection.setAttribute('data-visible', true);
+          newBookSection.setAttribute('data-visible', false);
+          listBookSection.setAttribute('data-visible', false);
+          break;
+        default:
+          break;
+      }
+    });
+  });
+}
+
+export function loadLocalStorage(bookInstanceFromBookStore) {
+  const bookStorage = JSON.parse(localStorage.getItem('bookCollection'));
+  if (bookStorage !== null) {
+    bookInstanceFromBookStore.store = bookStorage;
+  }
+}
+
+export function loadTime() {
+  const date = DateTime.now();
+  const timeHtml = document.querySelector('.time');
+
```

Comment:
- [ ] The keyword `function` is depreciated in the new javascript ES6 syntax and is replaced by the arrow functions. So I would suggest you to use  `arrow functions` instead of function keyword as well. 

**Example snippet:**

```js
export const loadTime = ()  => {
...
}

```